test/llvm_targets.zig

```diff
@@ -259,6 +260,7 @@ const targets = [_]std.Target.Query{
     .{ .cpu_arch = .thumb, .os_tag = .linux, .abi = .musleabihf },
     .{ .cpu_arch = .thumb, .os_tag = .rtems, .abi = .eabi },
     .{ .cpu_arch = .thumb, .os_tag = .rtems, .abi = .eabihf },
+    .{ .cpu_arch = .thumb, .os_tag = .vita, .abi = .eabihf },
```

Comment:
@vesim987 does the Vita support pure Thumb programs? For context, see discussion here: https://github.com/ziglang/zig/pull/24938#discussion_r2290934625

Reply:
I'd like to clarify that I'm not very deep into `vita` homebrew but it seems that unlike the `3ds`, the `thumb` ISA is the default when compiling for it.

Sources: [vita's gcc buildscript](https://github.com/vitasdk/buildscripts/blob/2aeb041d227176f695eaf203e7e93c012b307f78/CMakeLists.txt#L451), [rust's vita target docs](https://doc.rust-lang.org/rustc/platform-support/armv7-sony-vita-newlibeabihf.html#requirements)

Reply:
Well then:

* Does it support binaries that contain Arm start code? Or must the start code be Thumb?
* Does it support programs switching between Arm and Thumb modes at will?